### DISCUSSION TITLE
Assign an issue to a user

### DISCUSSION THREAD

**Discussion by juanppdev** [community/community#148212] [Actions]
### Select Topic Area

General

### Body

Good evening, I am trying to make an actions for a kanban project in a project, what I want is to be able to use github actions and for example: I create an issue and assign it to someone, then in the kanban the issue moves to “In Progress” automatically, but I have not been able to do it.

**Comment by AyoubMabrouk** [comment]
Hey, I usually use github workflow to automate actions : 
1- Create a Workflow File
2- Write the .yaml code (according to your needs)
3- If you wanna schedule it each period of time, add a con command to your yaml code, for example : 

on:
  schedule:
    # Runs every 5 hours
    - cron: '0 */5 * * *'
    
4- commit it.

I hope this could help :)

**Comment by juanppdev** [comment]
Good morning, this is one of the codes I've been using, but it didn't work.

name: Mover issue asignado a "En proceso"
on:
  issues:
    types:
      - assigned
jobs:
  move-assigned-issue:
    runs-on: ubuntu-latest
    steps:
      - name: Mover a "En proceso"
        uses: alex-page/github-project-automation-plus@v0.2.0
        with:
          project: "proyectoPrueba"  # Verify that the project name is correct
          column: 'En proceso'      # Verify that the column exists and is called exactly 'In Process'.
          repo-token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}  # Verifica que el token tenga los permisos adecuados
